client/client-multi/client-c/src/kaa_event.c

```diff
@@ -495,6 +576,37 @@ static kaa_error_t kaa_event_list_serialize(kaa_event_manager_t *self, kaa_list_
     return KAA_ERR_NONE;
 }
 
+static kaa_error_t kaa_event_listeners_request_serialize(kaa_event_manager_t *self, kaa_platform_message_writer_t *writer, uint16_t *listeners_count)
+{
+    uint16_t count = 0;
```

Comment:
This function is for internal needs. There is no need to double check arguments.